lib/Runtime/Library/TypedArray.cpp

```diff
@@ -2742,7 +2489,7 @@ namespace Js
             defaultConstructor = scriptContext->GetLibrary()->GetFloat32ArrayConstructor();
             break;
         case TypeId::TypeIds_Float64Array:
-            defaultConstructor = scriptContext->GetLibrary()->GetFloat32ArrayConstructor();
+            defaultConstructor = scriptContext->GetLibrary()->GetFloat64ArrayConstructor();
```

Comment:
>GetFloat64 [](start = 62, length = 10)

Wow, good catch!
(is there a unit test to guard any future regression, e.g. due to merge?)